Gemfile

```diff
@@ -58,6 +58,7 @@ group :job do
   gem 'qu-redis', require: false
   gem 'delayed_job_active_record', require: false
   gem 'sequel', require: false
+  gem 'celluloid', require: false
```

Comment:
Definitely don't need it for Sucker Punch anymore. The old version is dependent on Celluloid, but the gem would drag that in. Don't think it needs to be in here.